pyunfold/priors.py

```diff
@@ -5,10 +5,22 @@
 import pandas as pd
 
 
+def uniform_prior(norm, num_causes):
+    """Uniform Prior
+    """
+    # All bins are given equal probability.
+    # Most generic prior.
+    prior = norm * np.ones(num_causes) / num_causes
```

Comment:
I could be missing something, but I don't see why `norm` is needed here since the `prior` array gets normalized to 1 later.

I think something like 

```python
def uniform_prior(num_causes):
    '''Docstring
    '''
    prior = np.full(num_causes, 1/num_causes)
    return prior
```

is equivalent without `norm`.

Reply:
Yes, that's totally true. I was just keeping the format of before. Cause technically, the priors don't need to be normalized. This is because we are using them as prior count distributions, not prior probability distributions. After all, they are assigned as n_c for the first iteration. So I'll remove the norms from them and commit.

Reply:
The other option is to make them normalized in the prior.py file, and multiply them by the total number of observed counts when assigning to n_c just before starting the unfolding process.

Reply:
Ok, so what I did was remove norm from both prior functions, instead opting to scale any and all priors by the total number of observed counts.